apps/play/src/components/Sidebar.tsx

```
@@ -72,57 +25,19 @@ const Sidebar: Component<SidebarProps> = (props) => {
-            )}
-          </For>
-        )}
-
-        {samples().length === 0 && !loading() && (
-          <div class='no-samples'>No saved samples</div>
-        )}
-      </div>
+      {props.title && (
+        <div class='sidebar-header'>
+          <h3>{props.title}</h3>
+          <button class='close-button' onclick={props.onClose}>
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)